spec/volcanic/authenticator/v1/token_spec.rb

```diff
@@ -80,9 +81,30 @@
   end
 
   describe '#revoke' do
-    context 'When success' do
-      before { token.new(mock_token_base64_2).revoke! }
-      it { expect(token.new(mock_token_base64_2).remote_validate).to be false }
+    let(:token_params) {}
+    let(:token_instance) { token.new(token_params) }
+    let(:response) { { 'messages': 'token destroyed successfully' } }
+
+    subject { token_instance.revoke! }
+
```

Comment:
these tests are missing some very key details...

If I change the implementation to send the request to /api/nothing/exists/at/tokens this would still work.
If I change the implementation of the `claims_payload` method to return `{foo: bar}` this will generate a completely different checksum to the one needed by the server... so would fail, but these tests will still pass
There is no testing of what happens if the `Token` instance is created by passing in a hash of claims vs a string